modules/objc/examples/code_snippets/GettingStarted.m

```diff
@@ -13,28 +22,31 @@ @implementation GettingStarted
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
-    // Get the database (and create it if it doesn’t exist).
-
-    bool *testReplication;
-    testReplication = false;
-
+    
+    [self getStartedWithReplication: false];
+    
+}
+- (void) getStartedWithReplication: (Boolean) replication {
```

Comment:
What is this method for? Do we need any doc tags?

Reply:
for iOS no, the whole file is a snippet. All GettingStarted platforms specify that replication is OPTIONAL, although there is no optional in place for that piece of code in any other platform. Plan is to have another PR with those changes as it will be easier to review by the team - no specific iOS code in that one.